packages/rich-text-types/src/nodeTypes.ts

```diff
@@ -198,7 +198,7 @@ export interface ResourceHyperlink extends Inline {
 }
 
 export interface TableCell extends Block {
-  nodeType: BLOCKS.TABLE_HEADER_CELL | BLOCKS.TABLE_CELL;
+  nodeType: BLOCKS.TABLE_HEADER_CELL;
```

Comment:
We should instead keep `BLOCKS.TABLE_CELL` and remove header cell

Reply:
I believe having the two values there is a bug that went unnoticed for a quite a while

Reply:
We should also change the header row below content attribute to be `(TableCell | TableHeader)[]` instead of just `TableCell[]`